assets/src/blocks/frontendIndex.js

```diff
@@ -0,0 +1,24 @@
+import {GuestBookFrontend} from './GuestBook/GuestBookFrontend';
+
+// Render React components
+const COMPONENTS = {
+  'planet4-blocks/guestbook': GuestBookFrontend,
+};
+
+document.addEventListener('DOMContentLoaded', () => {
+  document.querySelectorAll('[data-render]').forEach(
+    blockNode => {
+      const blockName = blockNode.dataset.render;
+      if (!COMPONENTS[blockName]) {
+        return;
+      }
+
+      const BlockFrontend = COMPONENTS[blockName];
+      if (!BlockFrontend) {
+        return;
+      }
+      const attributes = JSON.parse(blockNode.dataset.attributes);
+      wp.element.render(<BlockFrontend {...attributes.attributes} />, blockNode);
```

Comment:
Correct me if I'm wrong, but I think `hydrate` instead and then change `date-render` to `data-hydrate` to all the occurrences.

Reply:
As discussed on DM with @GP-Dan-Tovbein , we will follow up on this change, whenever [PLANET-6917](https://jira.greenpeace.org/browse/PLANET-6917) is merged.